File: app/src/main/java/ca/unb/mobiledev/ultimatestattracker/model/Player.java
package ca.unb.mobiledev.ultimatestattracker.model;

import java.io.Serializable;

public class Player implements Serializable {
    public String fName;
    public String lName;
    public int number;
    public GENDER gender;

    public enum GENDER {
        Male,
        Female,
        Other
    }

    public Player(String fName, String lName, int num, GENDER gender){
        this.fName = fName;
        this.lName = lName;
        this.number = num;
        this.gender = gender;
    }
}

File: app/src/main/java/ca/unb/mobiledev/ultimatestattracker/model/Player.kt
package ca.unb.mobiledev.ultimatestattracker.model

class Player : java.io.Serializable {
    fun getFormattedName(): String {
        return "[$number] $fName $lName"
    }

    var fName: String
    var lName: String
    var number: Int
    var gender: GENDER

    constructor(fName: String, lName: String, number: Int, gender: GENDER){
        this.fName = fName
        this.lName = lName
        this.number = number
        this.gender = gender
    }
    enum class GENDER {
        Male, Female, Other
    }

    override fun toString(): String {
        return getFormattedName()
    }
}